ui/src/app/app.component.ts

```diff
@@ -1,4 +1,16 @@
 import { Component } from '@angular/core';
+import { faCoffee } from '@fortawesome/free-solid-svg-icons';
```

Comment:
You can do 
`import { faCoffee, faShoppingBa } from '@fortawesome/free-solid-svg-icons' `